crt_portal/cts_forms/templates/forms/question_cards/date.html

```diff
@@ -13,5 +13,5 @@
   </p>
   {% endif %}
 
- {% include 'forms/snippets/date_components.html' %}
+ {% include 'forms/snippets/date_components.html' with required=True %}
```

Comment:
The date card is shared, so this does add the * on the form/new page

Reply:
Thanks for the catch! Fixed.